anda/tools/nerdfetch/nerdfetch-tools.spec

```
@@ -0,0 +1,37 @@
+Name: nerdfetch
+Version: 8.4.0
+Release: 1%?dist
+Summary: A POSIX *nix fetch script using Nerdfonts 
+
+License: MIT
+URL: https://github.com/ThatOneCalculator/NerdFetch
+Source0: %url/archive/refs/tags/v%version.tar.gz
+
+BuildArch: noarch
+Requires: /usr/bin/sh
+
```

Since this requires Nerd Font symbols it can dep on the symbols (which we carry).
```suggestion
Requires: nerdfontssymbolsonly-nerd-fonts

```

Alternatively (can be changed to Requires but this package installs *every* Nerd font):
```suggestion
Suggests: nerd-fonts

```